AuthSAML.php

```diff
@@ -266,6 +269,14 @@ public function newUserSession()
         }
         $flag = $this->get('simplesamlphp_cookie_session_storage', null, null, true);
         if ($flag) {
+            /* if class already exist : use it, else call
+             * @see https://github.com/auth-it-center/Limesurvey-SAML-Authentication/pull/6#discussion_r1529844452
+             **/
+            if (class_exists('SimpleSAML_Session')) {
+                $session = SimpleSAML_Session::getSessionFromRequest();
+            } else {
+                $session = \SimpleSAML\Session::getSessionFromRequest();
+            }
             $session = \SimpleSAML\Session::getSessionFromRequest();
```

Comment:
This line (280) should be removed as it ignores the if block

Reply:
Arg … sorry …

Reply:
> This line (280) should be removed as it ignores the if block

fixed